components/event-system/queue.js

```diff
@@ -0,0 +1,89 @@
+const HOW_MANY_PRIORITIES_ARE_THERE_NITRO = 6
+const eventQueue = Array.from(Array(HOW_MANY_PRIORITIES_ARE_THERE_NITRO), () => [])
+const eventContentListeners = []
+let eventContentOnce = []
+
+function eventQueueEmpty () {
+  for (const queue of eventQueue) {
+    if (queue.length) {
+      return false
+    }
+  }
+  return true
+}
+
+function notifyOfContent () {
+  for (const callback of eventContentListeners) {
+    try {
+      callback()
+    } catch (_e) {
```

Comment:
You got around the single character variable names by adding an underscore. 🤦🏻‍♂️

Insted, rewrite these with variable names that make sense. In this case, just `error` would be fine.